docker-app/qfieldcloud/core/models.py

```diff
@@ -1415,7 +1421,65 @@ class Meta:
         managed = False
 
 
-class Delta(models.Model):
+class AbstractValidatedProjectOwner(models.Model):
+    """ "
+    Assumes a foreign key field 'project' for a Project
+    """
+
+    class Meta:
+        abstract = True
+
```

Comment:
```suggestion

    class Meta:
        abstract = True

```

This is no longer needed.